packages/client/src/pages/CheckerPage.js

```diff
@@ -160,13 +161,25 @@ const CheckerPage = ({ checker, matomoTrackEvent, resetChecker, topic }) => {
     if (Number.isInteger(value)) {
       // Edit specific question index (value), go directly to this new question index
       newQuestionIndex = value;
+      if (!checker.stack[newQuestionIndex]) {
+        captureException(
+          `Go to question, question with index: ${newQuestionIndex} not found`
+        );
+        return null;
```

Comment:
It could just `return;` instead of `return null;`, because it's a plain function and not a component.